jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogStreamActor.kt

```diff
@@ -155,8 +158,9 @@ class LogStreamFilterActor(
         return getSearchLogEvents(request)
     }
 
-    private suspend fun getSearchLogEvents(request: FilterLogEventsRequest): List<LogStreamEntry> {
-        val response = client.filterLogEvents(request).asDeferred().await()
+    private fun getSearchLogEvents(request: FilterLogEventsRequest): List<LogStreamEntry> {
+        // withContext is required to give the execution the same scope as us so if it throws and exception we can catch it
```

Comment:
Comment seems old and not related to new code

Reply:
That comment was dangling from when it was using the async client, fixed